src/Model/Gallery.php

```diff
@@ -50,7 +50,8 @@ abstract class Gallery implements GalleryInterface
     protected $defaultFormat = MediaProviderInterface::FORMAT_REFERENCE;
 
     /**
-     * @var GalleryItemInterface[]|Collection
+     * @var Collection
+     * @phpstan-var Collection<GalleryItemInterface>
```

Comment:
```suggestion
     * @phpstan-var Collection<array-key, GalleryItemInterface>
```

And @phpstan-var is not really needed, phpstorm understand this. `Foo<bar>`

Reply:
Fixed